test/integration/reasoner/planner/PlannerTest.java

```diff
@@ -97,7 +124,7 @@ public void test_planner_retrievable_dependent_upon_concludable() {
         Retrievable retrievable = new Retrievable(resolvedConjunction("{ $c($b); }", logicMgr));
 
         Set<Resolvable<?>> resolvables = set(concludable, retrievable);
-        List<Resolvable<?>> plan = Planner.plan(resolvables, new HashMap<>(), set());
+        List<Resolvable<?>> plan = ReasonerPlannerTestWrapper.planResolvables(resolvables, set());
```

Comment:
then here to do the planning you ask the reasoner to do the planning:  `tx().reasoner().planner().planResolvables(...)`